src/less/lightbox-dialog/lightbox-dialog.less

```diff
@@ -18,6 +18,7 @@
     .dialog-body-content();
 
     min-height: 18px;
+    overflow: auto;
```

Comment:
It looks like the scrolling lightbox already existed in the docs/stories so it's not new, but this seems to be introducing it. Maybe we should add a story for it?

Reply:
Will do